src/main/java/com/ericsson/eiffel/remrem/publish/service/PublishResultItem.java

```diff
@@ -0,0 +1,54 @@
+package com.ericsson.eiffel.remrem.publish.service;
+
+public class PublishResultItem {
+
+    private String id;
+    private int status_code;
```

Comment:
There usually is an annotation that can be used to define the serialized name of a property to avoid having snake case variable names in Java. For gson it seems to be `@SerializedName("status_code")`